ocs_ci/ocs/ocp.py

```diff
@@ -928,6 +928,20 @@ def get_build():
     return ocp_cluster.get()['items'][0]['status']['desired']['version']
 
 
+def get_platform():
```

Comment:
You can use from node.py the get_provider() method.

Reply:
I've removed this function now.